src/components/Widgets/SubscriptionsWidget.tsx

```diff
@@ -55,7 +53,6 @@ const SubsWidget = () => {
     <div className="subscription-inventory">
       {isCardDataEmpty ? (
         <EmptyState variant={EmptyStateVariant.lg}>
-          <EmptyStateIcon icon={EmptyStateSubscriptionsIcon} />
```

Comment:
Issue: We still need to render this icon - if the icon component is deprecated we need to render it in some other way. I'm sure there's either a new prop to support icons or some other component.